src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/stores/ListStore.js

```diff
@@ -228,24 +228,30 @@ export default class ListStore {
             return {};
         }
 
-        const schemaSettings = ListStore.getSchemaSetting(this.listKey, this.userSettingsKey);
-        const schema = this.schema;
+        const schemaSettings = ListStore.getSchemaSetting(this.listKey, this.userSettingsKey) || [];
```

Comment:
Because of the default value used here the below `if` will never be true. We can either keep the default value and ditch the `if` or remove the default value, both options are fine for me.